### ISSUE TITLE
Release 7.5 release notes

### ISSUE THREAD

**Issue by kbwagner** [openopps/OpenOppsTasks#141]
Waiting to determine if release will officially be Friday 1/11/19.

1) Katrina to get draft release notes from Brice and review
2) Katrina to send Debbie draft release notes
3) Debbie to review draft release notes and finalize
4) Debbie/Matt to post release notes to What's new section of help center sometime on the morning of 1/11/19

**Comment by dalenard21** [comment]
@kbwagner do you know what's going out in the release (if it goes)?  Just want to make sure if there are any help center updates we'll need to push out, other than the "what's news'.

**Comment by kbwagner** [comment]
@dalenard21 the biggest change that is visible is that when you click Create Opportunity you will have to select a target audience.  So yes we might need to look at help content and see if there are updates.

**Comment by dalenard21** [comment]
From Katrina, here's what's new:

Changes made January 11, 2018

We made the following changes: 
Enhancements: 
•	Continued enhancing the ability to have opportunities for communities.
o	Community administrators will now get an email to indicate that there are opportunities ready for them to review and approve
o	Community members will now receive an email when they have been added to a community
o	Opportunity creators can select their target audience to determine whether the opportunity will be tied to an agency or a community of users that may not all be part of the same agency
•	Started the foundation for search improvements that will be coming soon to increase the efficiency and speed of searching for opportunities
•	Imported data from USAJOBS to standardize our information
•	Continued work on our internship communities that will be coming soon including the ability to create new internships, preview the internship during creation process, and the viewing of an internship. 

Bug fixes:
We fixed:
•	Navigation disappearing on some pages
•	An incorrect link appearing in the draft opportunity created email 

@kbwagner I decided to remove the part about "continued work on our internship communities..." because the user won't see those changes and it may cause confusion.  I think we should highlight the internship work when we actually push it out.
